sklearn/ensemble/_bagging.py

```diff
@@ -252,9 +258,10 @@ def __init__(
         n_jobs=None,
         random_state=None,
         verbose=0,
+        base_estimator="deprecated",
     ):
         super().__init__(base_estimator=base_estimator, n_estimators=n_estimators)
-
+        self.estimator = estimator
```

Comment:
I think that the `BaseEnsemble` should introduce the `estimator` parameter and deprecate the `base_estimator`. In this way, we can share the deprecation for all meta-estimators inheriting from it.

We can always specialized in the child class if neeeded.